kokoro/continuous.bat

```diff
@@ -5,9 +5,6 @@ set PATH=%JAVA_HOME%\bin;%PATH%
 
 cd github/cloud-opensource-java
 
-mkdir %USERPROFILE%\.m2
-copy settings.xml %USERPROFILE%\.m2
```

Comment:
Should we remove settings.xml from this repo? It's not used, I think.

Reply:
It's being removed in this PR (the 3rd file)

Reply:
show missed it because github doesn't deletes :-)